compiler/tests/fixtures/compile_errors/call_event_with_wrong_types.fe

```diff
@@ -0,0 +1,7 @@
+contract Foo:
+    event MyEvent:
+        val_1: string100
+        val_2: u8
```

Comment:
for sanity it might be good to mark one of these as indexed

Reply:
Good point :+1: